### ISSUE TITLE
Yamllint pre-commit and direct invocation yield inconsistent results

### ISSUE THREAD

**Issue by cansjt** [adrienverge/yamllint#241]
When running yamllint manually, it finds more errors than when invoked by pre-commit. Here are the command lines used:
```
yamllint --config-file .yamllint .
yamllint --config-file .yamllint .
pre-commit run --all-files
pre-commit run --all-files yamllint
```

Not sure yet what is the cause of the issue. But when debugging, I found out that the hook is not passed any arguments ever. Even if I add `args:` value to the hook's configuration (as seen below).

```yaml
# .pre-commit-config.yaml
---
repos:
- repo: https://github.com/adrienverge/yamllint.git
  rev: 1.21.0
  hooks:
  - id: yamllint
    files: ^.*\.(yaml(lint)?|yml)$
    # Don't use exclude here, put it in .yamllint
    args:
    - '--config-file'
    - '.yamllint'
```
I obviously tried to scrub pre-comment and install again the latest version of everything. No luck.

**Comment by adrienverge** [comment]
Hi Nicolas, thanks for the report.

We don't support `pre-commit` here (by the way I think it was a bad idea to accept https://github.com/adrienverge/yamllint/blob/503bde9/.pre-commit-hooks.yaml in the repo).

Can you create a question on Stack Overflow or similar?

**Comment by ffalorjr** [comment]
Doesn't look like it fixes your issue, but I was in the middle of googling how to do what you are trying to do. Here is what i did to "disable" line-length check... Note: I haven't seen anywhere that says i can actually disable this check so i'm just changing the max acceptable length.

``` yaml
- repo: 'https://github.com/adrienverge/yamllint'
    rev: v1.20.0
    hooks:
      - id: 'yamllint'
        args: ['-d', '{extends: default, rules: {line-length: {max: 999}}}']
```

**Comment by cbows** [comment]
I am also experiencing this issue. Yamllint invoked by pre-commit does not respect arguments I pass in pre-commit config.

**Comment by adrienverge** [comment]
Thanks for confirming this.

Please note that `pre-commit` support directly from this repo will be dropped in a future version.
